packages/sdk/src/chainConfig/addresses.ts

```diff
@@ -62,6 +72,8 @@ const chainSpecificAddresses: ChainSpecificAddresses = {
     UNISWAP_V2_ROUTER: "0x10ED43C718714eb63d5aA57B78B54704E256024E",
     UNISWAP_V2_FACTORY: "0x985BcA32293A7A496300a48081947321177a86FD",
     PAIR_INIT_HASH: ethers.utils.hexlify("0x00fb7f630766e6a796048ea87d01acd3068e8ff67d078148a3fa3f4a84f69bd5"),
+    STABLE_TOKEN: moonbeamAssets.find((a) => a.symbol === assetSymbols.madUSDC)!.underlying,
```

Comment:
these are the Nomad assets -- we likely want to use the multichain's USDC

Reply:
I saw have still have a uniswap pair with WGLMR and should have worked for the liqui bots

Reply:
well, it would but there's nothing being borrowed there